com.unity.render-pipelines.universal/Runtime/Data/UniversalRenderPipelineAsset.cs

```diff
@@ -885,6 +903,15 @@ static void UpgradeAsset(UniversalRenderPipelineAsset asset)
 
                 asset.k_AssetPreviousVersion = 5;
             }
+
+            if (asset.k_AssetPreviousVersion < 6)
```

Comment:
Why an empty upgrade?

Reply:
Before version 7 it would trigger this call:
            if (k_AssetPreviousVersion != k_AssetVersion)
            {
                EditorApplication.delayCall += () => UpgradeAsset(this);
            }

However now that I added version 7 it is actually not needed.
Going to remove it